### PR TITLE
feat(device-selection): design tweaks

### PR DESCRIPTION
**Pull request by virtuacoplenny**
- Change copy
- Add labels to dropdowns
- Adjust color and positioning

### CODE REVIEW

File: lang/main.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
         "startVideoMuted": "Everyone starts hidden",
         "selectCamera": "Camera",
         "selectMic": "Microphone",
-        "selectAudioOutput": "Audio output",
+        "selectAudioOutput": "Speakers",
```
Comment (bbaldino) [inline lang/main.json]: audio output feels better than speakers to me here, i worry calling this speakers will be confusing

Comment (virtuacoplenny) [reply lang/main.json]: Design confirms "Speakers."

Comment (bbaldino) [reply lang/main.json]: we're gonna need more discussion on this.

Comment (virtuacoplenny) [reply lang/main.json]: Emil doesn't want it changed so we're not changing it. He can discuss with design and the copy writer if they bring it up.

**Comment by virtuacoplenny** [comment]
Force pushed to rebase onto latest master.

**Pull request merged**